sympy/solvers/ode/tests/test_systems.py

```diff
@@ -781,6 +919,86 @@ def test_linear_3eq_order1_type4_slow():
+            for s in sol]
+    sol = [Eq(x1(t), Subs(-2*C1*c2*r2*exp(e5s)/(c1*r1 + c1*r2 - c2*r2 + sqrt(e1s)) -
+                          2*C2*c2*r2*exp(e4s)/(c1*r1 + c1*r2 - c2*r2 - sqrt(e1s)) - 2*c2*r2*Piecewise((-e2s*i*\
+                            r2*exp(-e5s)/(c1*sqrt(e1s)*r1 + c1*sqrt(e1s)*r2 + c2*sqrt(e1s)*r2 + e1s),
+                            Ne(c1*sqrt(e1s)*r1 + c1*sqrt(e1s)*r2 + c2*sqrt(e1s)*r2 + e1s, 0)),
+                            (-i*r2*t/sqrt(e1s), True))*exp(e5s)/(c1*r1 + c1*r2 - c2*r2 + sqrt(e1s)) -
+                            2*c2*r2*Piecewise((-e2s*i*r2*exp(-e4s)/(-c1*sqrt(e1s)*r1 - c1*sqrt(e1s)*r2 -
+                            c2*sqrt(e1s)*r2 + e1s), Ne(c1*sqrt(e1s)*r1 + c1*sqrt(e1s)*r2 + c2*sqrt(e1s)*r2 - e1s, 0)),
+                            (i*r2*t/sqrt(e1s), True))*exp(e4s)/(c1*r1 + c1*r2 - c2*r2 - sqrt(e1s)),
+                            (e1s, e2s, e3s, e4s, e5s), (c1**2*r1**2 + 2*c1**2*r1*r2 + c1**2*r2**2 - 2*c1*c2*r1*r2 +
+                            2*c1*c2*r2**2 + c2**2*r2**2, 2*c1*c2*r1*r2, -1/(2*c2*r2) - 1/(2*c2*r1) - 1/(2*c1*r1),
+                            sqrt(e1s)*t/e2s + e3s*t, -sqrt(e1s)*t/e2s + e3s*t))),
+            Eq(x2(t), Subs(C1*exp(e5s) + C2*exp(e4s) + Piecewise((-e2s*i*r2*exp(-e4s)/(-c1*sqrt(e1s)*r1 - c1*sqrt(e1s)*r2
+                            - c2*sqrt(e1s)*r2 + e1s), Ne(c1*sqrt(e1s)*r1 + c1*sqrt(e1s)*r2 + c2*sqrt(e1s)*r2 - e1s, 0)),
+                            (i*r2*t/sqrt(e1s), True))*exp(e4s) + Piecewise((-e2s*i*r2*exp(-e5s)/(c1*sqrt(e1s)*r1 +
+                            c1*sqrt(e1s)*r2 + c2*sqrt(e1s)*r2 + e1s), Ne(c1*sqrt(e1s)*r1 + c1*sqrt(e1s)*r2 +
+                            c2*sqrt(e1s)*r2 + e1s, 0)), (-i*r2*t/sqrt(e1s), True))*exp(e5s), (e1s, e2s, e3s, e4s, e5s),
+                            (c1**2*r1**2 + 2*c1**2*r1*r2 + c1**2*r2**2 - 2*c1*c2*r1*r2 + 2*c1*c2*r2**2 + c2**2*r2**2,
+                            2*c1*c2*r1*r2, -1/(2*c2*r2) - 1/(2*c2*r1) - 1/(2*c1*r1), sqrt(e1s)*t/e2s + e3s*t, -sqrt(e1s)*t/e2s\
+                            + e3s*t)))]
+    assert sol == sol_
+    assert checksysodesol(eq, sol) == (True, [0, 0])
+
+
+@slow
+@XFAIL
+def test_linear_new_order1_type2_de_lorentz():
+    if ON_TRAVIS:
+        skip("Too slow for travis.")
+
+    m = Symbol("m", real=True)
+    q = Symbol("q", real=True)
+    t = Symbol("t", real=True)
+
+    e1, e2, e3 = symbols("e1:4", real=True)
+    b1, b2, b3 = symbols("b1:4", real=True)
+    v1, v2, v3 = symbols("v1:4", cls=Function, real=True)
+
+    eqs = [
+        -e1 * q + m * Derivative(v1(t), t) - q * (-b2 * v3(t) + b3 * v2(t)),
+        -e2 * q + m * Derivative(v2(t), t) - q * (b1 * v3(t) - b3 * v1(t)),
+        -e3 * q + m * Derivative(v3(t), t) - q * (-b1 * v2(t) + b2 * v1(t))
+    ]
+
+    sol = dsolve(eqs)
+    assert checksysodesol(eqs, sol) == (True, [0, 0, 0])
+
+
+@slow
+@XFAIL
+def test_linear_neq_order1_type2_fails_too_slow():
```

Comment:
Same comments here about slow/xfail. Which part is slow or fails? Is it dsolve or checksysodesol?

If we have many examples that fail checksysodesol then I think it would be good to open an issue with the examples.

Reply:
@oscarbenjamin Ok, I would split all the test cases of this structure.